src/libraries/Fuzzing/DotnetFuzzing/Fuzzers/HttpHeadersFuzzer.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Http.Headers;
+using System.Runtime.InteropServices;
+
+namespace DotnetFuzzing.Fuzzers;
+
+internal sealed class HttpHeadersFuzzer : IFuzzer
+{
+    public string BlameAlias => "mizupan";
```

Comment:
@MihaZupan given your username and 

> ```c#
> /// <summary>Alias to assign crash report work items to.</summary>
> string BlameAlias { get; }
> ```
should this be 
```suggestion
    public string BlameAlias => "MihaZupan";
```
? Or should this be even hard-coded here?

(same question on the other implementations of `IFuzzer`)

Reply:
I can clarify in the summary, but these work items are filed in the internal azdo org. The alias here is the Microsoft alias, not the GitHub username.